src/client.rs

```diff
@@ -286,9 +289,9 @@ impl RequestBuilder {
             self.headers.set(Accept::star());
         }
         if self.client.auto_ungzip.load(Ordering::Relaxed) &&
-            !self.headers.has::<AcceptEncoding>() &&
-            !self.headers.has::<Range>() {
-            self.headers.set(AcceptEncoding(vec![qitem(Encoding::Gzip)]));
+           !self.headers.has::<AcceptEncoding>() && !self.headers.has::<Range>() {
+            self.headers
+                .set(AcceptEncoding(vec![qitem(Encoding::Gzip)]));
```

Comment:
Yes. The default is for chained calls where the total line length is >60 to be wrapped (see [chain_one_line_max](https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#chain_one_line_max))